packages/ozone/src/db/schema/setting.ts

```diff
@@ -0,0 +1,24 @@
+import { Generated } from 'kysely'
+import { Member } from './member'
+
+export const settingTableName = 'setting'
+
+export type SettingScope = 'personal' | 'instance'
+
+export interface Setting {
+  id: Generated<number>
```

Comment:
May consider this if we never want it set from app code.
```suggestion
  id: GeneratedAlways<number>
```